af/oometa/013-ooexplr-meta.install.sql

```diff
@@ -0,0 +1,81 @@
+BEGIN;
+
+select _v.register_patch( '013-ooexplr-meta', ARRAY[ '012-sha256-input-uniq' ], NULL );
+
+CREATE TABLE ooexpl_recent_msmt_count AS
+SELECT
+COUNT(msm_no) as msmt_count,
+probe_cc,
+probe_asn,
+test_name,
+test_start_time,
+bucket_date
+FROM measurement
+JOIN report ON report.report_no = measurement.report_no
+JOIN autoclaved ON autoclaved.autoclaved_no = report.autoclaved_no
+WHERE test_start_time > current_date - interval '31 day'
+GROUP BY probe_cc, probe_asn, test_name, test_start_time, bucket_date;
+
+ALTER TABLE ooexpl_recent_msmt_count
+ADD PRIMARY KEY(probe_asn, probe_cc, test_name, test_start_time, bucket_date);
+
+CREATE INDEX "ooexpl_recent_msmt_count_probe_cc_idx" ON "public"."ooexpl_bucket_msmt_count"("probe_cc");
+CREATE INDEX "ooexpl_recent_msmt_count_test_name_idx" ON "public"."ooexpl_bucket_msmt_count"("test_name");
+CREATE INDEX "ooexpl_recent_msmt_count_test_start_time_idx" ON "public"."ooexpl_bucket_msmt_count"("test_start_time");
+
+CREATE TABLE ooexpl_bucket_msmt_count AS
+SELECT
+COUNT(msm_no) as count,
+probe_asn,
+probe_cc,
+bucket_date
+FROM measurement 
+JOIN report ON report.report_no = measurement.report_no
+JOIN autoclaved ON autoclaved.autoclaved_no = report.autoclaved_no
+GROUP BY bucket_date, probe_asn, probe_cc;
```

Comment:
:bike: :derelict_house: &mdash; I'd suggest to use consistent naming patterns.
E.g. `msm` instead of `msmt` and `msm_count` instead of `msmt_count.
E.g. `msm_count` instead of `count`.

Reply:
Good point, I will look into fixing this.